rllib/utils/framework.py

```diff
@@ -9,6 +9,17 @@
 TensorType = Any
 
 
+def get_auto_framework():
+    """Returns a framework (str) to use automatically.
+
+    If only PyTorch is installed, returns "torch", otherwise "tf".
+    """
+    # PyTorch is installed.
+    if torch is not None and tf is None:
+        return "torch"
```

Comment:
This should probably raise an error instead if both are installed.

Reply:
+1